src/components/voyage/VoyageLogEntry.tsx

```diff
@@ -1,51 +1,54 @@
 import React from "react";
 import STTApi, { CONFIG } from "../../api";
-import { List, Image, Icon } from 'semantic-ui-react';
+import { Image, Icon } from 'semantic-ui-react';
 
 export class VoyageLogEntry extends React.Component<any,any> {
    constructor(props:any) {
       super(props);
 
       this.props.log.forEach((entry: any) => {
-         // TODO: some log entries have 2 crew
          if (entry.crew) {
-            let rc = STTApi.roster.find(rosterCrew => rosterCrew.symbol == entry.crew[0]);
-            if (rc) entry.crewIconUrl = rc.iconUrl;
+            entry.crewIconUrls = [];
+            for(let i = 0; i < entry.crew.length; i += 1) {
+               let rc = STTApi.roster.find(rosterCrew => rosterCrew.symbol == entry.crew[i]);
+               entry.crewIconUrls.push(rc ? rc.iconUrl ?? '' : '');
+            }
          }
       });
    }
 
    render() {
-      let listItems : any[] = [];
-      this.props.log.forEach((entry: any, index:number) => {
-         if (entry.crewIconUrl) {
-            listItems.push(
-               <List.Item key={index}>
-                  <Image avatar src={entry.crewIconUrl} />
-                  <List.Content>
-                     <List.Header>
-                        <span dangerouslySetInnerHTML={{ __html: entry.text }} />
-                     </List.Header>
-                     {entry.skill_check && (
-                        <List.Description>
-                           <span className='quest-mastery'>
-                              <img src={CONFIG.SPRITES['icon_' + entry.skill_check.skill].url} height={18} />
-                              {entry.skill_check.passed == true ? <Icon name='thumbs up' /> : <Icon name='thumbs down' />}
-                           </span>
-                        </List.Description>
-                     )}
-                  </List.Content>
-               </List.Item>
-            );
-         } else {
-            listItems.push(
-               <List.Item key={index}>
-                  <span dangerouslySetInnerHTML={{ __html: entry.text }} />
-               </List.Item>
+      const listItems : any[] = [];
+      this.props.log.forEach((entry: any, index: number) => {
+         const images = [];
+         if (entry.crewIconUrls) {
+            for(let j = 0; j < entry.crewIconUrls.length; j += 1) {
+               images.push(
+                  <Image avatar className='mini' src={entry.crewIconUrls[j]} key={`${index}-${j}`} />
+               );
+            }
+         }
+         let textClass = 'vle-text';
+         if (entry.skill_check && (index + 1) === this.props.log.length) {
+            textClass = `${textClass} ui label ${entry.skill_check.passed ? 'green' : 'red'}`
```

Comment:
Using "ui label green" or "ui label red" makes this look like a button, but does not function as one. Please adjust the styling here.